sihl-email/src/sihl_email.ml

```diff
@@ -180,7 +180,7 @@ module MakeSmtp (Config : SmtpConfig) : Sihl.Contract.Email.Sig = struct
   let stop () = Lwt.return ()
 
   let lifecycle =
-    Sihl.Container.create_lifecycle Sihl.Contract.Email.name ~start ~stop
+    Sihl.Container.create_lifecycle Sihl.Contract.Email.name_smtp ~start ~stop
```

Comment:
@aronerben The contract is not allowed to know implementation details such as SMTP.

Reply:
Done